src/modules/wes/actions.js

```diff
@@ -21,10 +21,18 @@ export const fetchRuns = networkAction(() => (dispatch, getState) => ({
     err: "Error fetching WES runs",
 }));
 
-export const receiveRunDetails = (runID, data) => ({
+/**
+ * Manually dispatch a run details receive event, equivalent to what is fired by the network action.
+ * @param {string} runID
+ * @param {Object} data
+ * @param {number | undefined} ts
+ * @return {{data, runID, type: string, ts: number | undefined}}
+ */
+export const receiveRunDetails = (runID, data, ts = undefined) => ({
```

Comment:
Minor: I find `ts` to be ambiguous for an function argument name, it would be more clear if we name it `timestamp` I think.